src/doc/rustc/src/platform-support/arm-linux.md

```diff
@@ -70,7 +70,7 @@ Targets that start with `armv4t` are for processors implementing the Armv4T
 architecture from 1994. These include the ARM7TDMI, as found in the Nokia 6110
 brick-phone and the Game Boy Advance. The 'T' stands for *Thumb* and indicate
 that the processors can execute smaller 16-bit versions of some of the 32-bit
-Arm instructions. Because a Thumb is like a small version of an Arm.
+Arm instructions. A Thumb is like a small version of an Arm.
```

Comment:
to nitpick (but since the entire PR is nitpicks I think that's fine :D): This is not an improvement, I think it makes the text worse

Reply:
https://github.com/tshepang/rust/commit/9aeab7807602c24685314e8c0f4e0a33e5ad5818